ui/src/widgets/ui-table/UITable.vue

```diff
@@ -183,7 +192,41 @@ export default {
                 action: 'multi_select'
             }
             this.$socket.emit('widget-action', this.id, msg)
+        },
+        updateIsMobile () {
+            this.isMobile = this.checkBreakpoint()
+        },
+        checkBreakpoint () {
+            let table = this.$refs.table
+            if (!table) {
+                return false
+            }
+            table = table.$el
+            const width = table.clientWidth
```

Comment:
@joepavitt I've noticed that the actual device screen sizes are not aligning with defined breakpoints.

Therefore I think this should be `window.innerWidth` or `screen.width` instead as this breakpoint checks the table's width even the ui table properties show mobile, table and desktop.

Also when it comes to css media queries, it checks browser window width as usual.

Available breakpoints 

<img width="255" alt="Screenshot 2024-08-14 at 14 19 28" src="https://github.com/user-attachments/assets/237e950a-7c30-4bb2-a5c8-70c95be10831">

Selected breakpoint is 768px

<img width="499" alt="Screenshot 2024-08-14 at 14 18 35" src="https://github.com/user-attachments/assets/71d5de5b-2992-4531-8e2e-439d65141140">

But the table got re-arranged at a different breakpoint

<img width="788" alt="Screenshot 2024-08-14 at 14 22 16" src="https://github.com/user-attachments/assets/95e9f6ff-c63e-4e3b-8972-8af19aa7e18c">

Reply:
> I've noticed that the actual device screen sizes are not aligning with defined breakpoints.

This is because it's based on the table width, and is intentional, maybe the labelling/help needs to be clearer. If you have a table that is 50% the width of the screen, then I think we should be switching to card view if that table is too wide for it's space

Reply:
Got it, I understand the use case now. Switching to card view when the table is too wide for its space makes sense. Thanks for clarifying!

Reply:
I agree thought, that the docs/help should make this cleaer, so, I'll add that in before we merge

Reply:
Thanks, I'll hold off on this for now then